lib/GeometryMath.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+
+module.exports = {
+    sumBarycentric: sumBarycentric,
+    flattenTriangle: flattenTriangle,
+    pointLineDistanceParametric: pointLineDistanceParametric
+};
+
+// Functions for working with geometry like triangles and lines
+
+var sumBarycentricScratch = new Cartesian3();
+function sumBarycentric(barycentric, vector0, vector1, vector2, result) {
```

Comment:
But there are no Node build tools to optimize them out. Node code doesn't get combined/minified like browser code does and even if we did, we certainly wouldn't be using requirejs to do it (which is the only system that uses these pragmas).  Just like we don't leave TODOs or commented out code in master; we shouldn't leave these unused pragmas in either.

Even if we had some sort of Node build, server-side code is a different beast than client code and removing error checking is a bad idea.  I can understand your point about Math code, but as we've already seen in Cesium, NaNs still sneak in unexpectedly, especially when you are dealing with external data like the gltf pipeline will.  Server code should have more checks.

If for some reason we ever need the pragmas, it will be easy to find all of the places we are throwing developer error and add them back it; you could probably even do it with a regex.

Finally, I will add that if we want to perform extra error checking for non-production builds, the standard way to do this is by checking for the environment variable, `NODE_ENV=production`, and only run code if it's not set.  But that doesn't make sense for most developer errors, since you're just replacing one if check with another; but it can still be useful for performance-intensive error checks.